app/src/main/java/de/westnordost/streetcomplete/quests/pitch_lit/PitchLit.kt

```diff
@@ -0,0 +1,10 @@
+package de.westnordost.streetcomplete.quests.pitch_lit
+
+enum class PitchLit(val osmValue: String) {
+    NIGHT_AND_DAY("24/7"),
+    AUTOMATIC("automatic"),
```

Comment:
I don't think these two are reasonable values for this.

24/7 is meant for for example lighting in tunnels (or indoors). But with the filter above, you already filter out any pitches that may be indoor.

automatic doesn't really make sense for pitches: It turns on when something moves? Oh, better not standon the tennis court, otherwise the lights go off again!

So, this could be a Boolean. Yes, no

Reply:
Yes, that makes sense. I removed those values.